{"name": "test_account_linking_upto_1000", "status": "skipped", "statusDetails": {"message": "Skipped: Skipping test due to unexpected login error: WebDriverException", "trace": "('C:\\\\Users\\\\usmaan.arshad\\\\PycharmProjects\\\\PythonProject\\\\tests\\\\Load_Records_Mass_Actions\\\\test_account_linking_upto_1000.py', 54, 'Skipped: Skipping test due to unexpected login error: WebDriverException')"}, "start": 1745315314536, "stop": 1745315413868, "uuid": "c046a8af-6f2e-4f10-990e-02bf0a5315be", "historyId": "5747452d8fc432ba7812cf6c21e97cec", "testCaseId": "5747452d8fc432ba7812cf6c21e97cec", "fullName": "tests.Load_Records_Mass_Actions.test_account_linking_upto_1000#test_account_linking_upto_1000", "labels": [{"name": "feature", "value": "Mass Actions"}, {"name": "story", "value": "Test mass actions for linking of accounts for up to 1000 records."}, {"name": "severity", "value": "critical"}, {"name": "tag", "value": "P1"}, {"name": "tag", "value": "regression"}, {"name": "parentSuite", "value": "tests.Load_Records_Mass_Actions"}, {"name": "suite", "value": "test_account_linking_upto_1000"}, {"name": "host", "value": "RT-LT-597"}, {"name": "thread", "value": "21104-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.Load_Records_Mass_Actions.test_account_linking_upto_1000"}]}